src/components/PublicBeat.tsx

```diff
@@ -99,21 +132,20 @@ const PublicBeat: React.FC = () => {
       <div className={classes.scroll}>
         {beats.map((beat, key) => {
           return (
-            <div className={classes.card} key={key}>
+            <div className={classes.card} key={key} onClick={() => playPublicBeat(beat.id)}>
               <img alt='' className={classes.background} src={beat.background}></img>
               <div className={classes.bottomLeftCorner}>
-                <img alt='' style={{float: 'left'}} className={classes.beatPicture} src={beat.picture}></img>
-                <div style={{float: 'left'}}>
+                <img className={classes.beatPicture} src={beat.picture} alt=""></img>
+                <div>
                   <div>{beat.title}</div>
-                  <div>
-                    <button style={{float: 'left'}}>Play Button</button>
-                    <div style={{float: 'left'}}>
+                  <div className={classes.playButtonAndBeatInfo}>
+                    <div>
                       <div>{beat.type}</div>
                       <div>{beat.duration}</div>
                     </div>
                   </div>
                   </div>
-                </div >
+                </div>
             </div>
           )
         })}
```

Comment:
I'm not sure but it looks like 4-spaced tabs and 2-spaced tabs are being mixed here